src/DocumentFormat.OpenXml/Packaging/OpenXmlPartContainer.cs

```diff
@@ -546,6 +546,24 @@ public OpenXmlPart GetPartById(string id)
             }
         }
 
+        /// <summary>
+        /// Try gets the child part through the relationship ID.
+        /// </summary>
+        /// <param name="id">The relationship ID of the part.</param>
+        /// <param name="part">The part.</param>
+        /// <returns>Return true when the part with the specified id exist, otherwise false</returns>
+        public bool TryGetPartById(string id, out OpenXmlPart part)
+        {
+            ThrowIfObjectDisposed();
+
+            if (id == null)
```

Comment:
use `id is null` - considered better practice now and slowly switch the code base over to that.